mlrun/datastore/targets.py

```diff
@@ -549,9 +549,7 @@ def write_dataframe(
                     os.makedirs(dir, exist_ok=True)
             target_df = df
             partition_cols = None  # single parquet file
-            if not target_path.endswith(".parquet") and not target_path.endswith(
-                ".pq"
-            ):  # directory
+            if not target_path.endswith((".parquet", ".pq")):  # directory
```

Comment:
Isn't it the same code as in `_is_artifact_data_single_file`? Maybe the method can be relocated to more suitable location so in can be used from both places

Reply:
I created new common function - `is_parquet_file`